includes/Admin/Notices/Manager.php

```diff
@@ -128,25 +128,25 @@ public function show_permalink_setting_notice( $notices ) {
     }
 
     /**
-     * Display single page product page create notice.
+     * Display dokan admin logo update notice.
      *
-     * @since 3.8.2
+     * @since DOKAN_SINCE
      *
      * @param array $notices
      *
      * @return array
      */
-    public function show_one_step_product_add_notice( $notices ) {
-        if ( 'yes' !== get_option( 'dismiss_dokan_one_step_product_nonce', 'no' ) ) {
+    public function show_admin_logo_update_notice( array $notices ): array {
+        if ( 'yes' !== get_option( 'dismiss_dokan_admin_logo_update_notice', 'no' ) ) {
             $notices[] = [
-                'type'        => 'alert',
-                'title'       => __( 'New One-Step Product Form', 'dokan-lite' ),
-                'description' => __( 'Try it now to enhance your vendor\'s product upload experience, the older two-step version will be retired in one month.', 'dokan-lite' ),
-                'priority'    => 1,
+                'priority'          => 1,
                 'show_close_button' => true,
-                'ajax_data'   => [
-                    'action' => 'dismiss_dokan_one_step_product_notice',
-                    'nonce'  => wp_create_nonce( 'dismiss_dokan_one_step_product_nonce' ),
+                'type'              => 'info',
+                'title'             => __( 'New Dokan Logo!', 'dokan-lite' ),
+                'description'       => __( 'Introducing the new freshly designed Dokan Logo. Keep an eye on your admin dashboard!!', 'dokan-lite' ),
```

Comment:
@jamil-mahmud vai, Did you check the notice message of Admin panel.

Reply:
@mrabbani vai
Title: Dokan came up with a new look!
Description: A new rebranded look is introduced in the entire platform. Check the updated visuals in different places. 

Recommending this change in the messages